MobSF/settings.py

```diff
@@ -345,12 +347,9 @@
     #==============================================
 
     #===========ANDROID EMULATOR SETTINGS ===========
-    # generated by mobsfy_AVD.py, do not edit the
-    # below AVD settings yourself.
-    AVD_EMULATOR = "avd_emulator"
-    AVD_PATH = "avd_path"
-    AVD_REFERENCE_NAME = r'Nexus5API16'
-    AVD_DUP_NAME = r'Nexus5API16_1'
+    # Android-Studio 'emulator' binary path
+    AVD_EMULATOR = "/Users/matandobr/Library/Android/sdk/tools/emulator"

```

Comment:
Do you want to provide this as a commented example and set the default to `""`

Reply:
Mmm doesn't matter, whatever you think is better (I check if the binary exists in the start_avd)